omnigibson/examples/teleoperation/robot_teleoperate_demo.py

```diff
@@ -1,28 +1,47 @@
 """
 Example script for using external devices to teleoperate a robot.
 """
+try:
+    from mediapipe import solutions
+except ModuleNotFoundError:
+    pass
+
 import omnigibson as og
 from omnigibson.utils.ui_utils import choose_from_options
+from omnigibson.utils.teleop_utils import TeleopSystem
+
+from telemoma.utils.camera_utils import RealSenseCamera
+from telemoma.configs.base_config import teleop_config
```

Comment:
Move all three of these into the main function so that we don't import telemoma at import time.